### PR TITLE
translated chapter 10 in romanian

### PR DESCRIPTION
**Pull request by aniakchak09**

### CODE REVIEW

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Delete the `.idea` folder from the repository.

**Comment by alexandruradovici** [comment]
Please send this to upstream and place a link to it here.

File: lessons/ro/chapter_10.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- title: Chapter 10 - Sfârșit
+  content_markdown: >
+    A fost o plăcere să te avem alături în Turul limbajului Rust. Ferris și echipa
+    Turului limbajului Rust speră din suflet că te vei bucura de ce îți rezervă viitorul! Dacă 
+
+    te-ai simțit bine până acum, îți recomandăm să continui cu resursele:
```
Comment (alexandruradovici) [inline lessons/ro/chapter_10.yaml]:
```suggestion
    ți-a plăcut acest tur, îți recomandăm să continui cu resursele:
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Send this pull request to upstream and place a link to it here.

**Comment by aniakchak09** [comment]
https://github.com/richardanaya/tour_of_rust/pull/496

**Pull request closed without merging**